rviz_visual_testing_framework/src/internal/executor.cpp

```diff
@@ -43,6 +43,11 @@ void Executor::queueAction(std::function<void(void)> action)
   increaseTotalDelay();
 }
 
+void Executor::wait(int milliseconds_to_wait)
```

Comment:
Recommendation: if this should never be negative (my assumption reading the code), then make it a `size_t`.